salt/modules/zcbuildout.py

```diff
@@ -1032,14 +1032,20 @@ def _check_onlyif_unless(onlyif, unless, directory, runas=None, env=()):
                 if not onlyif:
                     _valid(status, 'onlyif execution failed')
             elif isinstance(onlyif, string_types):
-                if retcode(onlyif, cwd=directory, runas=runas, env=env) != 0:
+                if retcode(
```

Comment:
Can we separate the zcbuildout update here into a separate PR since it functions in the module domain instead of the state domain? Making shell changes in modules introduces a separate security context then in the state system.